static/sass/styles.scss

```diff
@@ -639,4 +639,4 @@ summary {
 
 .is-bordered--top {
   @extend %vf-pseudo-border--top;
-}
```

Comment:
Again this is copied from base cve branch to avoid conflicts

Reply:
?This doesn't make sense.... regardless of rebasing, if there's a `}` in this branch that isn't in the `cve` branch then either CSS was broken in the `cve` branch or it's broken in this branch. I'll see if I can discover which it is.

Reply:
Oh my bad, it's simply a newline character.